frontend/projects/admin/src/app/event/event-dashboard/event-dashboard.component.html

```diff
@@ -3,6 +3,375 @@
+        <div class="col-2">
+          <img
+            src="/file/{{ eventOrganizationInfo.event.fileBlobId }}"
+            alt=""
+            class="img-fluid"
+          />
+        </div>
+        <div class="col-6">
+          <dl class="row">
+            <dt class="col-sm-3">Public URL</dt>
+            <dd class="col-sm-9">
+              <a
+                [href]="
+                  (instanceSetting$ | async)?.baseUrl +
+                  '/event/' +
+                  eventOrganizationInfo.event.shortName
+                "
+                target="_blank"
+                rel="noopener"
+                >{{
+                  (instanceSetting$ | async)?.baseUrl +
+                    "/event/" +
+                    eventOrganizationInfo.event.shortName
+                }}</a
+              >
+            </dd>
+            <dt class="col-sm-3">Organized by</dt>
+            <dd class="col-sm-9">
+              {{ eventOrganizationInfo.organization.name }} &lt;{{
+                eventOrganizationInfo.organization.email
+              }}&gt;
+            </dd>
+            <dt class="col-sm-3">Website URL</dt>
+            <dd class="col-sm-9">
+              <a
+                href="{{ eventOrganizationInfo.event.websiteUrl }}"
+                target="_blank"
+                rel="noopener"
+                >{{ eventOrganizationInfo.event.websiteUrl }}</a
+              >
+            </dd>
+            <dt class="col-sm-3">T&C URL</dt>
+            <dd class="col-sm-9">
+              <a
+                href="{{ eventOrganizationInfo.event.termsAndConditionsUrl }}"
+                target="_blank"
+                rel="noopener"
+                >{{ eventOrganizationInfo.event.termsAndConditionsUrl }}</a
+              >
+            </dd>
+            <dt
+              *ngIf="eventOrganizationInfo.event.privacyPolicyUrl"
+              class="col-sm-3"
+            >
+              Privacy Policy URL
+            </dt>
+            <dd
+              *ngIf="eventOrganizationInfo.event.privacyPolicyUrl"
+              class="col-sm-9"
+            >
+              <a
+                href="{{ eventOrganizationInfo.event.privacyPolicyUrl }}"
+                target="_blank"
+                rel="noopener"
+                >{{ eventOrganizationInfo.event.privacyPolicyUrl }}</a
+              >
+            </dd>
+            <dt *ngIf="!eventOrganizationInfo.event.online" class="col-sm-3">
+              Location
+            </dt>
+            <dd *ngIf="!eventOrganizationInfo.event.online" class="col-sm-9">
+              <a
+                href="{{ eventOrganizationInfo.event.location }}"
+                target="_blank"
+                rel="noopener"
+                >{{ eventOrganizationInfo.event.location }}</a
+              >
+            </dd>
+            <dt class="col-sm-3">Languages</dt>
+            <dd class="col-sm-9">
+              <ul class="breadcrumb language-breadcrumb mb-0">
+                <li
```

Comment:
&lt;li> should be marked as "breadcrumb-item", see https://getbootstrap.com/docs/5.2/components/breadcrumb/